heron/api/src/java/com/twitter/heron/api/Config.java

```diff
@@ -309,7 +309,8 @@ public static void setSerializationClassName(Map<String, Object> conf, String cl
 
   /**
    * Is topology running with acking enabled?
-   * @deprecated use {@link #setTopologyReliabilityMode()} instead.
+   * @deprecated use {{@link #setTopologyReliabilityMode(Map, TopologyReliabilityMode)}} 
```

Comment:
This occurrence should redirect people to the one with the similar constructor, so the first one with the map. The other javadoc error in this class should send them to the second one without it.